src/_events/eventsManifest.json

```diff
@@ -297,6 +297,17 @@
       "description": "mlh-localhost-learn-c-sharp-with-unity-feb-2019/description.md",
       "banner": "mlh-localhost-learn-c-sharp-with-unity-feb-2019/banner.jpg"
     },
+    {
```

Comment:
You put this in a weird place in the list but that's really just stylistic.

Reply:
It's not a huge deal. We should probably move away from a single manifest file, to one that generates in the build later.